sdk/purview/Azure.Analytics.Purview.Account/src/Generated/AccountsClient.cs

```diff
@@ -0,0 +1,507 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.Analytics.Purview.Account
+{
+    /// <summary> The Accounts service client. </summary>
+    public partial class AccountsClient
+    {
+        /// <summary> The HTTP pipeline for sending and receiving REST requests and responses. </summary>
+        public virtual HttpPipeline Pipeline { get; }
+        private readonly string[] AuthorizationScopes = { "https://purview.azure.net/.default" };
+        private readonly TokenCredential _tokenCredential;
+        private Uri endpoint;
+        private readonly string apiVersion;
+        private readonly ClientDiagnostics _clientDiagnostics;
+
+        /// <summary> Initializes a new instance of AccountsClient for mocking. </summary>
+        protected AccountsClient()
+        {
+        }
+
+        /// <summary> Initializes a new instance of AccountsClient. </summary>
+        /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
+        /// <param name="endpoint"> server parameter. </param>
+        /// <param name="options"> The options for configuring the client. </param>
+        public AccountsClient(TokenCredential credential, Uri endpoint = null, PurviewAccountClientOptions options = null)
```

Comment:
The shape of the generated constructor here is wrong.  It should be:

```csharp
AccountsClient(Uri endpoint, TokenCredential credential, PurviewAccountClientOptions options = null)
```

I think some changes to the x-ms-parameterized-host in the swagger will cause M4 to interpret this differently and fix the generation.  You can look at either Scanning or Catalog to see what we did there.

Longer term, we should figure out if we can make the defaults better so that the code generator does the right thing instead of requiring a very specific x-ms-parameterized-host value.